mobile-app/app/components/SummaryTitleV2.tsx

```diff
@@ -0,0 +1,121 @@
+import BigNumber from 'bignumber.js'
+import NumberFormat from 'react-number-format'
+import { tailwind } from '@tailwind'
+import { ThemedTextV2, ThemedViewV2 } from './themed'
+import { View } from '.'
+import { ViewProps } from 'react-native'
+import { getNativeIcon } from '@components/icons/assets'
+import { translate } from '@translations'
+import { RandomAvatar } from '@screens/AppNavigator/screens/Portfolio/components/RandomAvatar'
+
+type SummaryTitleProps = React.PropsWithChildren<ViewProps> & ISummaryTitleProps
+
+interface ISummaryTitleProps {
+  title: string
+  amount: BigNumber
+  testID: string
+  fromAddress: string
+  fromAddressLabel?: string | null
+  toAddress?: string
+  iconA: string
+  iconB?: string
+}
+
+export function SummaryTitleV2 (props: SummaryTitleProps): JSX.Element {
+  const IconA = getNativeIcon(props.iconA)
+  const IconB = props.iconB !== undefined ? getNativeIcon(props.iconB) : undefined
```

Comment:
```suggestion
  const IconB = props.iconB ?? getNativeIcon(props.iconB)
```

Reply:
the original logic should be correct because we should get native icon only when `props.iconB` is defined